common/static/common/templates/image-modal.underscore

```diff
@@ -3,7 +3,7 @@
     <%= smallHTML%>
     <a href="#" class="modal-ui-icon action-fullscreen" role="button">
       <span class="label">
-        <i class="icon fa fa-arrows-alt fa-large"></i> <%= gettext("Fullscreen") %>
+        <span class="icon fa fa-arrows-alt fa-large" aria-hidden="true"></span> <%= gettext("Fullscreen") %>
```

Comment:
While you are editing these lines, go ahead and correct the escaping (<%- instead of <%=).